bin/fun-local-invoke.js

```diff
@@ -28,6 +28,8 @@ program
     `a file containing event data passed to the function during
                              invoke, If this option is not specified, it defaults to
                              reading event from stdin`)
+  .option('--tmp-dir <tmpDir>', 'Custom mount directory.')
```

Comment:
The temp directory mounted to /tmp , default to './.fun/invoke-tmp/{service}/{function}/'

Reply:
done